chat-demo-android/src/main/kotlin/com/twilio/chat/demo/activities/ChannelActivity.kt

```diff
@@ -68,6 +70,8 @@ class ChannelActivity : Activity(), ChatClientListener, AnkoLogger {
                 finish()
             }
             R.id.action_unregistercm -> basicClient.unregisterFcmToken()
+            R.id.action_crash_in_chat_client -> basicClient.chatClient!!.simulateCrash(CHAT_CLIENT_CPP)
+            R.id.action_crash_in_tm_client -> basicClient.chatClient!!.simulateCrash(TM_CLIENT_CPP)
```

Comment:
TMClient is internal name which we will most probably rename to chat::Client at some point, better not codify it like this. Will raise brows in a couple years from now.

Reply:
`TMClient.cpp` is what user will see in the crash's title on crashlytics website. I don't see better options to name it for now and then try to keep it updated

Reply:
Ok then.